tests/Neo.UnitTests/IO/UT_MemoryReader.cs

```diff
@@ -49,5 +51,120 @@ public void TestReadNullableArray()
             var n = reader.ReadNullableArray<UInt256>();
             Assert.AreEqual(5, reader.Position);
         }
+
+        [TestMethod]
+        public void TestReadSByte()
+        {
+            sbyte value = -5;
+            byte[] byteArray = new byte[1];
+            byteArray[0] = (byte)value;
+            MemoryReader reader = new(byteArray);
+            var n = reader.ReadSByte();
+            n.Should().Be(value);
+        }
+
+        [TestMethod]
+        public void TestReadInt32()
+        {
+            int value = -5;
```

Comment:
when you do these test, can you please add more cases, such as int.min, int.max, int.min-1, int.max+1, 0, -1, 1.